src/clj/rems/db/user_mappings.clj

```diff
@@ -0,0 +1,46 @@
+(ns rems.db.user-mappings
+  (:require [clojure.tools.logging :as log]
+            [rems.db.core :as db]
+            [rems.json :as json]
+            [medley.core :refer [deep-merge]]
+            [schema.coerce :as coerce]
+            [schema.core :as s]))
+
+(s/defschema UserMappingsDataDb
+  {:from s/Str
+   :from-value s/Str
+   :to-value s/Str})
```

Comment:
the namings are quite ad-hoc. maybe we can come up with better ones? in my head, `from` is the value/attribute we want to use to get the target (`to-value`).

Reply:
renamed these, good suggestions :+1: